datasources/src/main/resources/datasources-index-mapping.yml

```diff
@@ -14,4 +14,6 @@ properties:
       keyword:
         type: keyword
   connector:
+    type: keyword
+  resultIndex:
```

Comment:
Do we need indexing on this?

Reply:
yeah, we need to save it and retrieve it later.

Reply:
keyword looks good, no full text search requuired.

Reply:
I have skipped properties because we are never going to query based on it. There is no harms in having it...in case some use case comes up.